mat64/dense.go

```diff
@@ -253,9 +254,10 @@ func (m *Dense) View(i, j, r, c int) Matrix {
 	return &t
 }
 
-// Grow returns an expanded copy of the receiver. The copy is expanded
-// by r rows and c columns. If the dimensions of the new copy are outside
-// the caps of the receiver a new allocation is made, otherwise not.
+// Grow returns the receiver expanded by r rows and c columns. If the dimensions
+// of the expanded matrix are outside the capacities of the receiver a new
+// allocation is made, otherwise not. Note the receiver itself is not modified
+// during the call to Grow.
```

Comment:
I definitely agree on the gonum-tour article set. It would also be a lot more fun to do than lapack translations and reviews :)

Yea, these are another of those weird cases. It's also odd that they don't behave like slices, i.e. View is only shorter. It's fine behavior for a matrix package, but it's something worth noting.